### PR TITLE
Remove setManagedClusterDefaultsv20170831

### PR DESCRIPTION
**Pull request by rjtsdl**
hcp scenario

**What this PR does / why we need it**:
The default value cause classicMode failure. The default value get introduced in #499

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by rjtsdl** [comment]
@dmitsh if you have already incorporated the change into your fix. You can close this, once your change is merged. :)

File: pkg/api/apiloader.go [hunk 1]
```diff
@@ -295,11 +294,6 @@ func setContainerServiceDefaultsv20170131(c *v20170131.ContainerService) {
 }
 
 // Sets default HostedMaster property values for any appropriate zero values
-func setManagedClusterDefaultsv20170831(hm *v20170831.ManagedCluster) {
-	hm.Properties.KubernetesVersion = ""
-}
-
-// Sets default HostedMaster property values for any appropriate zero values
 func setManagedClusterDefaultsvlabs(hm *apvlabs.ManagedCluster) {
```
Comment (amanohar) [inline pkg/api/apiloader.go]: Can you remove this as well?

Comment (rjtsdl) [reply pkg/api/apiloader.go]: Dmitry will have the change in his PR. Can you leave a comment to him. Yes, we should also remove this.

Comment (dmitsh) [reply pkg/api/apiloader.go]: closed in favor of #1578

**Pull request closed without merging**